src/test/java/fi/okm/jod/yksilo/externalapi/ExternalApiV1IntegrationTest.java

```diff
@@ -79,8 +86,38 @@ void getProfiilit400TooBigPageSize() {
         this.testRestTemplate.exchange(
             profiilitUrlWithParams,
             HttpMethod.GET,
-            new HttpEntity<>(new HttpHeaders()),
+            new HttpEntity<>(headersWithAuthentication()),
             new ParameterizedTypeReference<>() {});
     assertEquals(HttpStatusCode.valueOf(400), response.getStatusCode());
   }
+
+  @Sql(
+      scripts = {"/data/cleanup.sql", "/data/add_10_yksiloa.sql"},
+      executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
+  @Sql(
+      scripts = {"/data/cleanup.sql"},
+      executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD)
+  @Test
+  void unauthorized401IfNoApiKey() {
```

Comment:
tests should follow "should-expected-when-condition" naming scheme, e.g. here `shouldRespondUnauthorizedWhenNoApiKey`